sklearn not marked as a dependency, but needed to run the tests

If sklearn isn't installed, the mapeval tests will fail:

```
self = <toil_vg.test.test_vg.VGCGLTest testMethod=test_3_sim_small_mapeval>
outstore = '/scratch/tmp/tmpPxWYUu/toilvg-jenkinstest-outstore-a24a63f9-9b31-453f-bb8c-fbd1c6a824e3', count = 4000
names = ['vg', 'bwa-mem', 'bwa-mem-pe'], acc_threshold = 0.9

    def _assertMapEvalOutput(self, outstore, count, names, acc_threshold):
        with open(os.path.join(outstore, 'stats.tsv')) as stats:
            table = [line for line in stats]
        headers = set()
        for row in table[1:]:
            toks = row.split()
            self.assertTrue(len(toks) == 5)
>           name, count, acc, auc, qqr = toks[0], int(toks[1]), float(toks[2]), float(toks[3]), float(toks[4])
E           ValueError: could not convert string to float: sklearn_not_installed

src/toil_vg/test/test_vg.py:301: ValueError
```

However, `sklearn` isn't listed as a (dev) dependency, so it's possible to have all the dependencies and still have the tests fail.

--- a/version.py
+++ b/version.py
@@ -17,6 +17,7 @@ version = '1.2.1a1'
 required_versions = {'pyyaml': '>=3.11',
                      'boto3': '>=1.4.4',
                      'awscli': '>=1.10.1',
-                     'tsv': '==1.2'}
+                     'tsv': '==1.2',
+                     'scikit-learn': '==0.18.2'}
 
 dependency_links = []

